src/fairseq2/models/wav2vec2/asr/factory.py

```diff
@@ -58,7 +58,7 @@ class Wav2Vec2AsrConfig:
     temporal_mask_span_len: int = 10
     """The length of each temporal mask span that is applied over time steps."""
 
-    max_temporal_mask_prob: float = 0.70
+    max_temporal_mask_prob: float = 0.69
```

Comment:
Wondering if this change if just to use the default values from the paper or if there are other implications here.